_layouts/sidenav.html

```diff
@@ -1,27 +1,51 @@
 ---
 layout: main
 ---
+{% comment %}
+  This file implements a uniform way of maintaining translated sidenav lists. It will allow us to keep html out of markdown files. To add another page that uses side navigation, add the `sidenav` variable in the markdown file's front matter. Give it a unique value.
+    ---
+    sidenav: policies
+    ---
+
+  Add the new value to each i18n yaml file under the `nav` object. Define its sections with name/url pairs. The url value can be a relative url or hash anchor link (#).
+
+    policies:
+      sections:
+        - name: Our commitment to your privacy and security
+          url:  /policy/
+        - name: Another link
+          url: /policy/#another-policy-link
+  
+  The page.sidenav value will be passed to the sidenav for loop below.
+
+  Note: This does not apply to Help pages, which is far more complex.
+{% endcomment %}
 
 {% capture sidenav %}
-  <nav aria-label="{% t accessible_labels.secondary_navigation %}">
-    <ul class="usa-sidenav">
-      <li class="usa-sidenav__item">
-        <a href="/contact/" class="usa-current">{% t page.title %}</a>
-        <ul class="usa-sidenav__sublist">
-          {% for item in page.sidenav %}
-          <li class="usa-sidenav__item">
-            <a href="#{{ item.url }}">{{ item.name }}</a>
-          </li>
-          {% endfor %}
-        </ul>
-      </li>
-    </ul>
-  </nav>
+  {% if page.sidenav  %}
+    <nav aria-label="{% t accessible_labels.secondary_navigation %}" class="position-sticky pin-top">
+      <ul class="usa-sidenav">
+        <li class="usa-sidenav__item">
+          <!-- TODO: Parent item should not be clickable -->
+          <a href="{{ page.url }}" class="usa-current">{{ site.translations[site.lang]["nav"][page.sidenav]["title"] }}</a>
```

Comment:
I think that we're missing a translation? The accessibilty specs are complaining this is an empty link

```
    Expected the HTML found at $('.desktop\:display-none > .position-sticky.pin-top[aria-label="Secondary\ navigation"] > .usa-sidenav > .usa-sidenav__item > .usa-current[href$="accessibility\/"]') to have no violations:

    <a href="/accessibility/" class="usa-current"></a>
```

Reply:
Checks out now. Translation was in the other branch.